hotp/host/main.c

```diff
@@ -44,7 +44,7 @@ int main(void)
 	TEEC_Session sess;
 	TEEC_UUID uuid = TA_HOTP_UUID;
 
-	int i;
+	uint32_t i;
```

Comment:
`size_t` is more suitable than a fixed width integer.

Reply:
thanks.. updated